packages/veto-app/frontend/src/hooks/useSettings.ts

```diff
@@ -1,34 +1,23 @@
 import { useState, useEffect } from 'react'
-import gun from '../api/gun'
-
-export type Settings = {
-  darkmode?: boolean
-  passwordConfirm?: boolean
-}
+import { default as gun, settingsDb, Settings } from '@veto/db'
```

Comment:
Won't default be an array because `'@veto/db'` exports `packages/veto-app/shared/db/src/gun/index.ts` which export `src/gun/client.ts`?

I think I'm missing something :S

Reply:
It is because the import is "universal":
ex: file `shared/db/notifications.ts` can be imported in the frontend or the backend but client/server require a different initialization of gun